lib/handlers/formatting.js

```diff
@@ -47,16 +47,21 @@ export class FormattingHandler {
         }
         const versionArg = this.ceProps(`formatter.${formatter}.version`, '--version');
         const versionRe = this.ceProps(`formatter.${formatter}.versionRe`, '.*');
+        const hasExplicitVersion = this.ceProps(`formatter.${formatter}.explicitVersion`, '') !== '';
         try {
             const result = await exec.execute(exe, [versionArg], {});
             const match = result.stdout.match(versionRe);
             const formatterClass = getFormatterTypeByKey(type);
             const styleList = this.ceProps(`formatter.${formatter}.styles`);
             const styles = styleList === '' ? [] : styleList.split(':');
+            // If there is an explicit version, grab it. Otherwise try to filter the output
+            const version = hasExplicitVersion
```

Comment:
this could be expressed much simpler as:
```js
const version = this.ceProps(`formatter.${formatter}.explicitVersion`) || (match ? match[0] : result.stdout);
```

Reply:
Absolutely, thanks!